src/browser/commands/assert-view/capture-processors/assert-refs.js

```diff
@@ -25,5 +34,16 @@ exports.handleImageDiff = (currImg, refImg, state, opts) => {
         ...buildDiffOpts,
     };
 
-    return Promise.reject(ImageDiffError.create(state, currImg, refImg, diffOpts, diffAreas, diffBuffer));
+    return Promise.reject(
+        ImageDiffError.create({
```

Comment:
Changed this method's params to be a single object. This would be passed into html-reporter